### ISSUE TITLE
No facility for administrators to update public keys

### ISSUE THREAD

**Issue by lewellyn** [gitblit/gitblit#1052]
There is no way that I can find for an admin to view or update a user's public keys. There are a few valid scenarios for listing and updating the keys, such as for auditing or for administration/maintenance.

**Comment by gitblit** [comment]
True, there is no web ui for admins to mess with user SSH keys.  The assumption was administrators would have file access to the server and can manipulate the filesystem directly.
